internal/db/controllerproxy.go

```diff
@@ -1,36 +1,76 @@
+// Copyright 2025 Canonical.
+
 package db
 
 import (
 	"context"
+	"encoding/json"
 
 	"github.com/canonical/jimm/v3/internal/dbmodel"
 	"github.com/canonical/jimm/v3/internal/errors"
 	"github.com/canonical/jimm/v3/internal/servermon"
+	"github.com/juju/zaputil/zapctx"
+	"go.uber.org/zap"
+)
+
+const (
+	// These constant are used to create the appropriate identifiers for controller proxy data.
+	controllerProxyKind = "controllerProxy"
+	controllerProxyTag  = "controllerProxy"
+	typeKey             = "type"
+	configKey           = "config"
```

Comment:
These consts deserve better names so we know that they're for the controller proxy secret storage and avoid collisions.